src/scorecard.js

```diff
@@ -0,0 +1,28 @@
+class ScoreCard {
```

Comment:
With more time this could be split up into multiple classes. For example maybe you could create a Score class that's responsible for receiving and passing on user scores to ScoreCard.